### PR TITLE
My Index Page

### PR DESCRIPTION
**Pull request by LaocheXe**
Made it so the Index page would greet the USER or Guest

# Closes
This wasn't an issued #, sooooooooooooo yeah

# Changes Made
Describe your changes in detail here:
1. Edit the Index.razor page

# Notes
I updated my fork, applied changes, then Tried to run it, got some error's back about a missing .dll file - Basically - FiveOhFirstDataCore.Pages\bin\Debug\net6.0\ref\FiveOhFirstDataCore.Components.dll could not be found.

### CODE REVIEW

**Comment by Soyvolon** [comment]
In the future, when working on things that are not issues - make an issue for it first so it can be used for change tracking purposes.

File: FiveOhFirstDataCore/Pages/Index.razor [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
-General Kenobi.
+<AuthorizeView>
+ <Authorized>
+  <h1>Welcome back, @CurrentUser?.NickName!!</h1>
+ </Authorized>
+ <NotAuthorized>
+     <h1>Welcome Guest!</h1>
+ </NotAuthorized>
+</AuthorizeView>
+// Future Update, The Line Below will be setup for random quotes API
```
Comment (Soyvolon) [inline FiveOhFirstDataCore/Pages/Index.razor:12]: This is not a comment on a Razor page. Either use HTML comments or Blazor comments (`<!-- -->` or `@* *@`)

File: FiveOhFirstDataCore/Pages/Index.razor [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+    [CascadingParameter]
+    public Task<AuthenticationState> AuthStateTask { get; set; }
+
+    public bool InSquad { get; set; } = false;
+    public bool InPlatoon {get;set;} = false;
+    public bool InCompany { get; set; } = false;
+
+    private string Version { get; set; } = "";
+
+}
```
Comment (Soyvolon) [inline FiveOhFirstDataCore/Pages/Index.razor:32]: This should be moved to a code-behind page. Make an `Index.razor.cs` file (same directory as the `Index.razor` file) and create a partial index class to put this code into.

**Review by Soyvolon — CHANGES_REQUESTED** [review]

File: FiveOhFirstDataCore/Pages/Index.razor (hunk 1 above)
Comment (LaocheXe) [reply FiveOhFirstDataCore/Pages/Index.razor:12]: I blame php - it has me set up with //

**Comment by LaocheXe** [comment]
I hate the Visual Studio Git bull shit

**Pull request closed without merging**